packages/esm-patient-forms-app/src/config-schema.ts

```diff
@@ -72,20 +101,21 @@ export const configSchema = {
     _description: 'Whether to display recommended forms tab. Requires AMPATH ETL.',
     _default: false,
   },
-  showConfigurableForms: {
-    _type: Type.Boolean,
-    _description: 'Whether to use customURL to fetch forms, works with `customFormsUrl`',
-    _default: false,
-  },
   customFormsUrl: {
     _type: Type.String,
     _description: 'Custom forms endpoint to fetch forms using a custom url need `showConfigurableForms`',
     _default: '',
   },
-  orderFormsByName: {
+  useCurrentVisitDates: {
     _type: Type.Boolean,
-    _description: 'Order forms alphabetically.',
-    _default: true,
+    _description: 'Only load encounters inside of the current visits dated.',
+    _default: false,
+  },
+  orderBy: {
```

Comment:
Would be good to have a validator attached here that ensures the value is one of the two accepted values

Reply:
Hello @ibacher,
Can you be more specific on this one ?

Reply:
Goes with `_type` and `_description` below.